components/pipeline-service/development/kustomization.yaml

```diff
@@ -33,3 +33,12 @@ patches:
       namespace: tekton-results
       name: tekton-results-watcher
   - path: increase-results-pruner-gracePeriod.yaml
```

Comment:
@ramessesii2 @Roming22 - this was the line that was previously present that my yaml editor cited as a formatting error

Reply:
kustomize build worked fine even prior to your changes, since the yaml already defined the target properly. I'm fine with this change though. Thanks.